src/uct/ib/base/ib_iface.c

```diff
@@ -1770,23 +1799,27 @@ uct_ib_iface_estimate_perf(uct_iface_h iface, uct_perf_attr_t *perf_attr)
         break;
     default:
         send_pre_overhead        = iface_attr.overhead;
-        /* Doorbell write effect on CPU operations pipeline */
         send_post_overhead       = 40e-9;
-        /* Completion for every operation */
         send_post_overhead_zcopy = 20e-9;
     }
+    send_pre_overhead_bcopy = 5e-9;
+    send_overhead           = &ib_iface->config.send_overhead;
 
     if (perf_attr->field_mask & UCT_PERF_ATTR_FIELD_SEND_PRE_OVERHEAD) {
-        perf_attr->send_pre_overhead = send_pre_overhead;
+        perf_attr->send_pre_overhead = ucs_time_units_to_sec(send_overhead->pre,
```

Comment:
it seems like there are many "auto" values that are defined in the code
can't we write them explicitly as default values of the configuration, instead of using "auto"?

Reply:
In this case I need two somehow change values in case of CPU vendor is "Fujitsu ARM".
I am going to replace `auto` by the default values, after adding configuration file with the section for "Fujitsu ARM" case.

Reply:
maybe we can rebase this PR on the one that adds platform name to the parser?

Reply:
We should rebase on two PRs: the one that adds platform name to the parser (https://github.com/openucx/ucx/pull/9742), and the PR that adds configuration file with the section for "Fujitsu ARM" (will be added right after the previously mentioned PR).